easybuild/easyconfigs/p/picard/picard-2.0.1-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,20 @@
+name = 'picard'
+version = '2.0.1'
+
+homepage = 'http://broadinstitute.github.io/picard/'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM
+  (http://samtools.github.io/hts-specs) format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
please use `''` as version, otherwise the Java listed as a dep will not be loaded at build/install time (cfr. #2612)